src/services/store.ts

```diff
@@ -236,6 +236,22 @@ export function setLogsVisualizationType(type: string) {
   localStorage.setItem(VISUALIZATION_TYPE_LOCALSTORAGE_KEY, type);
 }
 
+// Line filter options
+const LINE_FILTER_OPTIONS_LOCALSTORAGE_KEY = `${pluginJson.id}.linefilter.option`;
+export function setLineFilterCase(caseSensitive: boolean) {
+  let storedValue = caseSensitive.toString();
+  if (typeof caseSensitive === 'boolean' && !caseSensitive) {
```

Comment:
Because of typescript, `typeof caseSensitive` will always be `boolean`, unless the types are incorrectly defined.